app/(overview)/page.tsx

```diff
@@ -1,7 +1,31 @@
+import CustomTable from "@/components/CustomTable/CustomTable";
+const DataTable = [
+  {
+    key: "1",
+    userName: "Mai Hồng Tín",
+    email: "tinhongmai@gmail.com",
+    creationDate: "2023-09-05",
+    event: "Event A",
+    team: "Team X",
+    role: "Leader",
+    flagSetting: true,
+  },
+  {
+    key: "2",
+    userName: "Admin",
+    email: "SuperAdmin@gmail.com",
+    creationDate: "2023-09-02",
+    event: "Event B",
+    role: "Admin",
+    flagSetting: true,
+  },
+];
+
 const Home = () => {
   return (
     <div>
       <h1 className="text-lg font-bold">Home</h1>
+      <CustomTable data={DataTable} />
```

Comment:
The property I mentioned above should be placed here. You should find a more meaningful name for it. `flagSetting` is not a good name.